lib/fig_newton/missing.rb

```diff
@@ -7,6 +7,7 @@ def method_missing(*args, &block)
       read_file unless @yml
       m = args.first
       value = @yml[m.to_s]
+      return value if type_bool?(value)
```

Comment:
I have an alternate solution that allows the tests to pass but raises the cyclomatic complexity of the method to unacceptable levels according to RuboCop: https://github.com/smoll/fig_newton/commit/2f5e3cc264db9e60527f467bcd96b3618ca1fb26#diff-27da7be4f06451c659e80246180dbfb7L12

Not really sure how to better rework this, though. I'm open to suggestions if anyone has any.